Point_set_3/include/CGAL/Point_set_3.h

```diff
@@ -619,11 +675,15 @@ class Point_set_3
   const_iterator garbage_end () const { return m_indices.end(); }
   /*!
     \brief Number of removed points.
+
+    \note The method `garbage_size()` is also available and does the
```

Comment:
what's the reason for having another function doing the same thing? At least does it need to be documented?

Reply:
Actually it is not documented while you claim it is in the small feature description. So you should probably not mention it.

Reply:
In my mind, it was the same reason why we have both `number_of_points()` and `size()` (see https://github.com/CGAL/cgal/blob/db0b7db315beeb4bfba91ded9e47d899bb991dff/Point_set_3/include/CGAL/Point_set_3.h#L269): the first one is for consistency with `Surface_mesh`, the second is for the `Range` concept.

Well, the `\note` is what I called "documenting" :)
It's documented the same as `size()` (although I agree that technically, `size()` is already documented through the `Range` concept).

If you think there's no point in documenting it at all, it's fine for me. It's just that I'm used to prefer the so-far undocumented version in my codes, and as there's a chance I end up using it in a public CGAL example by mistake, so I thought I'd better document it to make it clear.

Reply:
Then document it and remove it from the note.